libs/designsystem/src/lib/components/chart/chart.component.spec.ts

```diff
@@ -93,11 +118,16 @@ describe('ChartComponent', () => {
 
     Object.entries(scenarios).forEach(([property, { updateFn, newValue }]) => {
       describe(`${property}`, () => {
+        beforeEach(fakeAsync(() => {
+          spectator.component['ngAfterViewInit']();
+          tick();
+        }));
         it(`should update ${property}`, () => {
           const updateFnSpy = spyOn<any>(component, updateFn);
 
           spectator.setInput(property as any, newValue);
 
+          expect(spectator.component['chartHaveBeenRendered']).toBeTrue();
```

Comment:
```suggestion
```

Perhaps a bit too detailed for this test. 

If the following is true: 
`expect(updateFnSpy).toHaveBeenCalledTimes(1);` 

It means the test is successful no matter the state of `chartHasBeenRendered`. 

If `chartHasBeenRendered` is `false` the test will with the current implementation fail. But that is an implementation detail and not something i think we have to test for specifically 😃